src/keybaseca/config/config.go

```diff
@@ -143,13 +140,13 @@ func (ef *EnvConfig) GetKeybaseHomeDir() string {
 // Get the keybase paper key for the bot account. Used if you are running a separate instance of keybase for the chatbot.
 // May be empty.
 func (ef *EnvConfig) GetKeybasePaperKey() string {
-	return os.Getenv("KEYBASE_PAPERKEY")
+	return shared.ReadFileOrDefault(os.Getenv("KEYBASE_PAPERKEY_PATH"), os.Getenv("KEYBASE_PAPERKEY"))
 }
 
 // Get the keybase username for the bot account. Used if you are running a separate instance of keybase for the chatbot.
 // May be empty.
 func (ef *EnvConfig) GetKeybaseUsername() string {
-	return os.Getenv("KEYBASE_USERNAME")
+	return shared.ReadFileOrDefault(os.Getenv("KEYBASE_USERNAME"), os.Getenv("KEYBASE_USERNAME_PATH"))
```

Comment:
This is probably switched, isn't it?

Reply:
It does really look like that yes! Thank you for the notice! :)